tests/gordo_components/cli/test_cli.py

```diff
@@ -39,207 +32,199 @@
 logger = logging.getLogger(__name__)
 
 
-class CliTestCase(unittest.TestCase):
+@pytest.fixture
+def runner(tmp_dir):
```

Comment:
Is this required anytime we want to run some CLI tests against anything which would build the model with the CLI?

Reply:
Not *required*, but recommended. Else you'll end up with an `mlruns/` dir. Either that or you'd have to patch the context manager to use a mock MlflowClient.